t/27-consoles-s3270.t

```diff
@@ -77,8 +77,48 @@ subtest 's3270_console connect_and_login' => sub {
         [warnings { $s3270_console->connect_and_login(); }],
         bag(
             re(qr/connect_and_login.*\nRECONNECT.*\n/), re(qr/trying hard shutdown and reconnect.*/),
-        ), 'got reboot attempt',
+        ), 'reconnect attempt',
     );
+
+    $s3270_console_mock->redefine(expect_3270 => sub ($self, %arg) {
+            return ['Fill in your USERID and PASSWORD and press ENTER'] unless keys %arg == 1 && exists $arg{buffer_ready};
+            return ['RECONNECT'];
```

Comment:
Is it possible to mock `Sleep`? This return `RECONNECT` will cause `connect_and_login()` to sleep for 7 seconds as written here https://github.com/os-autoinst/os-autoinst/blob/master/consoles/s3270.pm#L377